Breeder/BR_EnvelopeUtil.cpp

```diff
@@ -1887,6 +1887,21 @@ bool BR_Envelope::FillProperties () const
 				{
 					m_properties.automationItems.push_back(WDL_FastString(token));
 				}
+				else if (strstr(token, "<EXT"))
+				{
+					int subchunks = 0; // <BIN, https://github.com/reaper-oss/sws/pull/1387#issuecomment-705651041
+
+					do
+					{
+						if (*token == '<')
+							++subchunks;
+						else if (!strcmp(token, ">"))
+							--subchunks;
+					
+							AppendLine(m_properties.EXT, token);
+							token = strtok(NULL, "\n");
```

Comment:
> How did you know about the BIN subchunk (never noticed/encountered it personally)?

The code was looking for a line with ">" to stop the loop so I wondered what would happen if the data contained that. REAPER inserted a <BIN subchunk when I tried "foo\n>".

---

Indentation fix:
```suggestion
						AppendLine(m_properties.EXT, token);
						token = strtok(NULL, "\n");
```

Reply:
Can I make a new dedicated commit for this on the `sws-2-12-1` branch?
Otherwise I wouldn't know how to fix it there because I've already merged other PR's afterwards (maybe I've been a bit too fast with merging..)

Reply:
I also pushed a new version of PR #1352 after it was merged. If you don't mind, I can try amending the 2.12.1 branch's history to replace the commits with the newer ones from those two PRs.

Reply:
That would be nice, I never figured out how to alter/amend 'in between' (i.e. not the latest) commit properly.
Thanks.